setup.py

```diff
@@ -39,9 +39,9 @@
         'six',
         'pandas>=0.20.3',
         'appdirs',
-        'tensorflow>=2.2.0',
+         # 'tensorflow',
```

Comment:
I think better to leave the `>=2.2.0` in the commented line just as documentation